### PR TITLE
First version of a unified init script.

### PR DESCRIPTION
**Pull request by jlingema**
Hi,

I added a first version of a global init script that could be used by all repositories. 

For example we could put it in the afs software area. Maybe @cbernet can comment whether that would also work for his typical MacOS workflow.

Cheers,
Joschka

### CODE REVIEW

**Comment by cbernet** [comment]
@hegner this one :-) I think we should probably updated this script with new LCG and ROOT

**Comment by jlingema** [comment]
Hi @cbernet, @hegner,

I updated to LCG_83 here as well. If we want to use this as the global init script, some small changes to FindPythia8 in FCCSW have to be made. I'll open a PR there in a minute.

Cheers,
Joschka

File: init_fcc_stack.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        else
+            echo "Take podio: $PODIO"
+        fi
+        if [ -z "$FCCEDM" ]; then
+            export FCCEDM=$FCCSWPATH/fcc-edm/0.3/x86_64-slc6-gcc49-opt
+        else
+            echo "Take fcc-edm: $FCCEDM"
+        fi
+        if [ -z "$FCCPHYSICS" ]; then
+            export FCCPHYSICS=$FCCSWPATH/fcc-physics/snapshot/x86_64-slc6-gcc49-opt
```
Comment (jlingema) [inline init_fcc_stack.sh]: We'll have to change this once we tag fcc-physics.

**Comment by jlingema** [comment]
From my point of view this is ready. If somebody could test, we can start using it for users that want to use standalone tools on afs. 

As mentioned above: If we want to use it in FCCSW we need to first merge HEP-FCC/FCCSW#79.

**Comment by cbernet** [comment]
Hi Joschka, thanks a lot! 
Did a fast test: 
- Could you please add $FCCPHYSICS/bin to the PATH? one needs to be able to find fcc-pythia8-generate which is a pretty useful tool
- I get the following error:

```
[test]$ /afs/cern.ch/exp/fcc/sw/0.7/fcc-physics/0.1/x86_64-slc6-gcc49-opt/bin/fcc-pythia8-generate ../../fcc 
physics/pythia8/ee_ZH_Zmumu_Hbb.txt 
start processing
output file: ee_ZH_Zmumu_Hbb.root
 PYTHIA Error in Settings::flag: unknown key Tune:preferLHAPDF
 PYTHIA Abort from Pythia::Pythia: unmatched version numbers : in code 8.175 but in XML 8.212
 PYTHIA Abort from Pythia::init: constructur initialization failed  

 *** Break *** segmentation violation

```

**Comment by jlingema** [comment]
Hi Colin,

Thanks for testing. I added fcc-physics to the path and fixed the installation. Could you check again? 

Thank you!
Joschka

**Comment by cbernet** [comment]
I still have the segmentation fault.
must go now. 

Cheers! 
Colin

**Comment by cbernet** [comment]
Hi @jlingema, 
For some reason, I don't have the segmentation fault anymore when running fcc-pythia8-generate. No idea why it occured in the first place, sorry about that. 

But then I get an issue later in heppy (see below). 

I tried to install fcc-physics myself instead of relying on the package you installed on afs, by doing: 

> unset FCCPHYSICS
> source ./init.sh 

In my copy of the fcc-physics package, and then it works. 
So there seems to be something fishy with the global install.
I'll be happy to do further testing, let me know. 

Cheers, 

Colin 

> [test]$ ipython -i analysis_ee_ZH_cfg.py 0 
> Python 2.7.9 (default, Nov 24 2015, 00:10:53) 
> Type "copyright", "credits" or "license" for more information.
> 
> IPython 4.0.1 -- An enhanced Interactive Python.
> ?         -> Introduction and overview of IPython's features.
> %quickref -> Quick reference.
> help      -> Python's own help system.
> object?   -> Details about 'object', use 'object??' for extra details.
> Error in cling::AutoloadingVisitor::InsertIntoAutoloadingState:
>    Missing FileEntry for /build/jlingema/build_dir/fcc-physics/tools/JetClusterizer.h
>    requested to autoload type JetClusterizer
> fccphysicsmItoolsDict dictionary payload:7:10: fatal error: '/build/jlingema/build_dir/fcc-physics/tools/JetClusterizer.h'
>       file not found
> #include "/build/jlingema/build_dir/fcc-physics/tools/JetClusterizer.h"
>          ^
> Error in TInterpreter::AutoParse: Error parsing payload code for class JetClusterizer with content:
> 
> #line 1 "fccphysicsmItoolsDict dictionary payload"
> 
> #ifndef G__VECTOR_HAS_CLASS_ITERATOR
>   #define G__VECTOR_HAS_CLASS_ITERATOR 1
> #endif
> 
> #define _BACKWARD_BACKWARD_WARNING_H
> #include "/build/jlingema/build_dir/fcc-physics/tools/JetClusterizer.h"
> 
> #undef  _BACKWARD_BACKWARD_WARNING_H
> 
> Error in TClass::LoadClassInfo: no interpreter information for class JetClusterizer is available even though it has a TClass initialization routine.

**Comment by hegner** [comment]
Ah. I know how to fix that.... it's a ROOT 6 oddity.

**Comment by jlingema** [comment]
Hi again,

It took us a bit to hunt down why the installed headers are not being picked up. I re-installed fcc-physics and kept the same version as nothing in the code itself has changed. In PR HEP-FCC/fcc-physics#10 are the needed changes in the CMakeLists.

I tested on lxplus and it works for me now.

Cheers,
Joschka

**Pull request merged**